webpack.config.js

```diff
@@ -25,7 +25,7 @@ const autoprefixer = require('autoprefixer');
 
 const env = process.env.NODE_ENV;
 
-const THIS_APP_ID = 'orderentry';
+const THIS_APP_ID = 'openmrs-owa-orderentry';
```

Comment:
@dessHub, why did you change the name to `openmrs-owa-orderentry` ?

Reply:
I can't remember changing it, but I guess it was a mistake. Will take care of it.